azurelinuxagent/common/conf.py

```diff
@@ -137,7 +137,8 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
     "Debug.CgroupDisableOnProcessCheckFailure": True,
     "Debug.CgroupDisableOnQuotaCheckFailure": True,
     "Debug.EnableFastTrack": True,
-    "Debug.EnableGAVersioning": False
+    "Debug.EnableGAVersioning": False,
+    "Debug.LogSystemWideMetrics": False
```

Comment:
let's try to minimize those debug options... is this one really needed or can you just use CgroupLogMetrics?